source/règles/base.yaml

```
@@ -1657,9 +1657,19 @@
 
 - espace: contrat salarié
   nom: coût du travail
```

La notion de coût est très orientée employeur. `Prix du travail` serait plus neutre économiquement. Quitte à laisser `coût total` pour le titre.
```suggestion
  nom: prix du travail
```

---

J'ai mis à jour sur la PR concernée #703